Dockerfile

```diff
@@ -1,74 +1,71 @@
 FROM ubuntu:14.04
```

Comment:
works for me, but i use other [cleanup](https://hub.docker.com/r/cgswong/min-jessie/) process
```
apt-get -y clean && apt-get -y autoclean && apt-get -y autoremove && \
cp -R /usr/share/locale/en\@* /tmp/ && rm -rf /usr/share/locale/* && mv /tmp/en\@* /usr/share/locale/ && \
rm -rf /var/cache/debconf/*-old && rm -rf /var/lib/apt/lists/* && rm -rf /usr/share/doc/*
```